lib/zoom/actions/user.rb

```diff
@@ -128,9 +128,10 @@ def user_token_delete(*args)
       end
 
       def user_email_check(*args)
-        # TODO: implement user_email_check
-        # params = Zoom::Params.new(Utils.extract_options!(args))
-        raise Zoom::NotImplemented, 'user_email_check is not yet implemented'
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.permit(:email)
+        params.require(:email)
```

Comment:
These methods can actually be chained because `.require` returns an instance of `Zoom::Parameters`

```suggestion
        params.require(:email).permit(:email)
```